bots/images/scripts/atomic.bootstrap

```diff
@@ -65,7 +65,7 @@ fi
 # will grow the partitions etc as appropriate, and atomic.setup will
 # explicitly grow the docker pool.
 
-vsize=$(qemu-img info "$out" --output=json | python -c 'import json, sys; print json.load(sys.stdin)["virtual-size"]')
+vsize=$(qemu-img info "$out" --output=json | python3 -c 'import json, sys; print json.load(sys.stdin)["virtual-size"]')
```

Comment:
Thanks! FTR, to make this independent of Python one can also use `jq '."virtual-size"'`